_events/sr2023/kickstart.md

```diff
@@ -1,19 +1,43 @@
 ---
 title: Kickstart
+date: 2023-10-22 10:00:00 Europe/London
+end_date: 2023-10-22 16:00:00 Europe/London
 layout: event
 type: kickstart
-location: Various
+location: Southampton University
 ---
 
 Kickstart is the event which kicks off a competition year. The structure of the
 competition, the game rules will be announced, kits will be handed out and of
 course any questions you have will be answered.
 
-Kickstart will be in person at a number of venues near teams. The date is
-expected to be on a Saturday at the end of October and will be announced closer
-to the time.
+If you are unable to attend Kickstart, please [let us know][teams-contact] so we
+can arrange another way to get you your kit.
 
-There's still time to [sign up][sign-up] if you have't already!
-Deadline for entries is Friday 30th September 2022.
+We'll have some desktops available for students to use, however bringing your
+own devices is highly recommended.
 
-[sign-up]: {{ site.baseurl }}/compete/
+## Directions
+
+The Southampton kickstart is being held at the University of Southampton at:
+
+Building 46,
+University Road,
+Southampton,
+SO17 1BJ
```

Comment:
```suggestion
Building 46,<br>
University Road,<br>
Southampton,<br>
SO17 1BJ<br>
```

In theory there's a way of doing this with trailling whitespace, however trailing whitespace is removed by many editors so that approach is painful and IMO using `<br>` is clearer.